packages/html-manager/src/libembed-amd.ts

```diff
@@ -3,6 +3,14 @@
 
 import * as libembed from './libembed';
 
+let cdn = 'https://unpkg.com/';
+
+// find the data-cdn for any script tag, assuming it is only used for embed-amd.js
+const scripts = document.getElementsByTagName('script');
+Array.prototype.forEach.call(scripts, (script) => {
```

Comment:
The standard trick here is to just look at the *last* script tag, which should be the one that loaded this script. I'm not sure how doing an async script tag affects this, though.

On the other hand, it looks like `currentScript` is a thing: https://stackoverflow.com/questions/403967/how-may-i-reference-the-script-tag-that-loaded-the-currently-executing-script

currentScript is supported on all the browsers that we care about: https://developer.mozilla.org/en-US/docs/Web/API/Document/currentScript

Reply:
Some comments on that stack overflow post seem to indicate that the `currentScript` sometimes doesn't work, but the last script does? Maybe we implement currentScript and if doesn't work in all cases, we fall back to the last script tag?

Reply:
By the time this code snippet gets executed, document.currentScript is null and all script tags are already all there (I guess it is after document.onload, already deep in require's callstack). So it is more robust to go over all script tags. Maybe avoid a name collision by renaming the attribute to `data-jupyter-widgets-cdn` ?

Reply:
FYI, i'm copying requirejs' behaviour with its data-main attribute, I guess they also have a reason for not trying to be smarter.

Reply:
I have no idea how using webpack+requirejs document.currentScript can ever be used, so I'd go with the `data-jupyter-widgets-cdn`.

Reply:
@jasongrout Yes, that looks good to me.